SCClassLibrary/Common/Collections/Dictionary.sc

```diff
@@ -438,9 +438,9 @@ IdentityDictionary : Dictionary {
 		index = this.scanFor(key);
 		array.put(index+1, value);
 		if ( array.at(index).isNil, {
-			array.put(index, key);
-			size = size + 1;
-			if (array.size < (size * 4), { this.grow });
+		array.put(index, key);
```

Comment:
maybe you can undo this reformatting (it is so annoying that autoindent doesn't leave comments alone)

Reply:
Sorry I thought I got these. I've un-done all of them in the next.